test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -31,20 +31,22 @@
 
 (deftest ^:integration legacy-routes
   (testing "The legacy web routing service properly handles old routes."
-    (bootstrap/with-puppetserver-running app
-      {}
-      (is (= 200 (:status (http-get "/v2.0/environments"))))
-      (is (= 200 (:status (http-get "/production/node/localhost"))))
-      (is (= 200 (:status (http-get "/production/certificate_statuses/all")))))))
+    (bootstrap/with-puppetserver-running-with-mock-jrubies
+     app
+     {}
+     (is (= 200 (:status (http-get "/v2.0/environments"))))
+     (is (= 200 (:status (http-get "/production/node/localhost"))))
+     (is (= 200 (:status (http-get "/production/certificate_statuses/all")))))))
```

Comment:
At least, simmons would cover the endpoints that a Puppet 3 agent would hit during its run.  I don't think an agent would hit the "environments" or "certificate_statuses" endpoints here.  For that matter, I guess "certificate_statuses" is pure Clojure anyway so that one's not even relevant to this whole discussion.